judge-d-server/src/test/integration/com/hltech/judged/server/infrastructure/persistence/environment/JPAEnvironmentRepositoryIT.groovy

```diff
@@ -54,6 +54,35 @@ class JPAEnvironmentRepositoryIT extends Specification {
             }
     }
 
+    def 'should overwrite what was persisted before - one environment with multiple spaces'() {
```

Comment:
According to test name it should check if `persist` method overwrite previous environment state so there is missing `persist` call in given section (look for test above)

Reply:
wrong test name - will fix it